vendor/github.com/containers/buildah/release.sh

```diff
@@ -64,26 +71,35 @@ release_commit()
 	write_spec_version "${VERSION}" &&
 	write_spec_changelog "${VERSION}" &&
 	write_changelog &&
-	git commit -asm "Bump to v${VERSION}"
+	git commit -asm "Bump to v${VERSION}
+
+[NO TESTS NEEDED]
```

Comment:
Why is this being added? The new-test checker is very careful not to require new tests on vendor-only changes.

Reply:
This PR just picks up all buildah changes. If this is wrong you have to fix this in buildah.

Reply:
@edsantiago  I don't think i added that manually. It was added automatically on vendor sync maybe from https://github.com/containers/buildah/pull/3516 @rhatdan any clue on this.

Reply:
@edsantiago let me know if this is a blocker I think extra `[NO TESTS NEEDED]` should not harm on version bump. But i am not aware of actual reasons behind it.